### ISSUE TITLE
[FEATURE]: Adding a scaffold generator

### ISSUE THREAD

**Issue by anikethsaha** [webpack/webpack-cli#802]
**Is your feature request related to a problem? Please describe.**
Creating a custom scaffold for a team or individual can be quite difficult if can't get a first stage help from the project 

**Describe the solution you'd like**
This generator like the other generators (loaders, plugins etc )  will certainly give a boilerplate code to the developer with the official scaffold guide's standards and support. And a developer can just work on top of that may be like adding custom few more questions and then use it for development. 

**Describe alternatives you've considered**
The same above

**Comment by rishabh3112** [comment]
Hi @anikethsaha, I wrote one [here](https://github.com/rishabh3112/webpack-scaffold-starter)  and published today only. It is not tested by users for now. Will be happy to get yours feedback 👍.

**Comment by misterdev** [comment]
Hi @anikethsaha, that's a good idea!

Good job @rishabh3112 😉 I'm testing your generator on windows but I'm getting an error:

![culo](https://user-images.githubusercontent.com/11232797/54840705-da8a1580-4ccd-11e9-96fa-e4f11b81cef9.png)

PS: if you prefer we can move this into an issue in your repo

**Comment by anikethsaha** [comment]
@rishabh3112  yup I will give feedback soon over that. I added a PR in your repo in order to solve the error mentioned by @misterdev.

**Comment by anikethsaha** [comment]
@rishabh3112 yup I will give feedback soon over that. I added a PR in your repo in order to solve the error mentioned by @misterdev.

**Comment by rishabh3112** [comment]
@misterdev have a look now. Have shipped the fix at v1.0.3.
Thanks for PR @anikethsaha :)

**Comment by misterdev** [comment]
That's great! What about adding to the README the instructions to use it without installing `webpack-cli init webpack-scaffold-starter`? 

PS: @rishabh3112 @anikethsaha, if you want, I would appreciate feeback on #799 :)

**Comment by anikethsaha** [comment]
@misterdev That article is very well written. good work on that

**Comment by rishabh3112** [comment]
@misterdev which `README` ? You mean at `webpack-cli`?
If so, I am waiting a review on this by @webpack/cli-team. If they affirm we can add this as a recommendation for starting a new scaffold 😄

**Comment by anikethsaha** [comment]
@rishabh3112 I think @misterdev meant in your project's readme to test/run it locally

**Comment by anikethsaha** [comment]
@rishabh3112 @misterdev Can you guys please test [this](https://bitbucket.org/anikethsaha/scaffold-generator/src/master/). I would really appreciate your feedback

**Comment by misterdev** [comment]
@anikethsaha it's nice, I generally think that in order to be used it should provide as many detailed instructions as possible on how to set up and run everything :)

**Comment by evenstensberg** [comment]
@anikethsaha could you work on this?

**Comment by anikethsaha** [comment]
Yeah sure! 
I guess me and @rishabh3112 can work on this as we planned earlier.

**Comment by rishabh3112** [comment]
Yes, This issue has been resolved and package is released to npm.
Project is available at https://github.com/rishabh3112/webpack-scaffold-starter.

**Comment by evenstensberg** [comment]
Okay great 👍
